docs/plugins/stonesense.rst

```diff
@@ -56,6 +56,16 @@ views, fog, and rotation. Here's the important section:
    :literal:
    :end-before: VALID ACTIONS:
 
+
+Streaming stonesense
+--------------------
+If you wish to stream stonesense thru OBS then you may find that opening
+stonesense causes your main DF window to flicker between DF and stonesense.
+While it is unknown exactly what causes this, a fix does exist. Simply make
+sure that both DF and stonesense are using ``Window Capture``s and NOT
+``Game Capture``s. This will stop the flickering from happening and enable
```

Comment:
if you want text directly after quoted text, you have to escape it. e.g.
```md
``Game Capture``\s
```

However, I think this would read better without the "s"s at all -- ```
"... are using ``Window Capture`` and NOT ``Game Capture``.
```

I think having the period directly after it is ok and doesn't need to be escaped.

Reply:
ahh ok think for both. i think i shall remove the s